components/common/typescript.gni

```diff
@@ -152,4 +153,4 @@ template("pack_web_resources") {
                             "visibility",
                           ])
   }
-}
\ No newline at end of file
+}
```

Comment:
do we need new line here?

Reply:
I think this is just standard eof newline. My editor automatically adds it and his probably does too